spec/fixtures/modules/yaml/plans/upload.yaml

```diff
@@ -7,9 +7,8 @@ steps:
     source: 'yaml/test.sh'
     destination: '/tmp/test_upload.sh'
     target: $nodes
-    arguments: ["foo", "bar", "baz"]
   - name: cleanup
-    eval: rm '/tmp/test_upload.sh'
+    command: rm '/tmp/test_upload.sh'
```

Comment:
I changed this with the following assumptions: 
1. Upload step does accept arguments
2. Eval does not accept targets

I think these were just copy paste errors?